rDNA/R/rDNA.R

```diff
@@ -1806,4 +1807,289 @@ print.dna_barplot <- function(x, trim = 30, attr = TRUE, ...) {
   x2$Entity <- sapply(x2$Entity, function(e) if (nchar(e) > trim) paste0(substr(e, 1, trim - 1), "*") else e)
   class(x2) <- "data.frame"
   print(x2)
+}
+
+#' Plot dna_barplot object.
+#'
+#' Plot barplots generated from dna_barplot.
+#'
+#' This function plots agreement and disagreement towards DNA Statements for
+#' different entities such as, e.g., "concept", "organization" or "person".
+#' 
+#' @param object A dna_barplot object.
+#' @param lab.pos,lab.neg Names for (dis-)agreement labels.
+#' @param lab Should (dis-)agreement labels and title be displayed?
+#' @param colors If \code{TRUE}, attribute colors will be used to fill the
+#'   bars.
+#' @param fontSize Text size in pts.
+#' @param barWidth Thickness of the bars. Bars will touch when set to \code{1}.
+#'   When set to \code{0.5}, space between two bars is the same as thickness of
+#'   bars.
+#' @param axisWidth Thickness of the x-axis which separates agreement from
+#'   disagreement.
+#' @param truncate Sets the number of characters to which axis labels should be truncated.
+#' @param exclude.min Reduces the plot to entities with a minimum frequency of statements.
+#'
+#' @examples
+#' \dontrun{
+#' dna_init()
+#' dna_sample()
+#'
+#' dna_openDatabase("sample.dna", coderId = 1, coderPassword = "sample")
+#' 
+#' # compute barplot data
+#' b <- dna_barplot(statementType = "DNA Statement",
+#'                  variable = "concept",
+#'                  qualifier = "agreement")
+#' 
+#' # plot barplot with ggplot2                
+#' autoplot(b)
```

Comment:
Can you write unit tests for the different usage scenarios of the function? The idea is that the person who writes a function usually has some code examples to try out the new function anyway (for example, you probably have an example with integer values), and that makes it easy to add those tests right away.